### PR TITLE
feat: Add new File Browser template to Dokploy

### PR DESCRIPTION
**Pull request by xmahbub**
### feat: Add new File Browser template to Dokploy

- Introduced a new template for File Browser, allowing users to deploy a web-based file management interface.
- File Browser enables file upload, deletion, preview, renaming, and editing within a specified directory.
- Supports multi-user functionality, allowing each user to have a unique directory.
- This template can be deployed as a standalone app, making it versatile for different file management use cases.

#### Why:
The addition of the File Browser template provides users with an intuitive file management solution that is accessible via a web interface, enhancing flexibility and functionality in managing files within Docker-based environments.

#### Impact:
- Simplifies deployment for users needing a web-based file manager.
- Enhances user experience by offering a powerful, customizable file management tool.
- Enables secure, multi-user access to designated directories for specific workflows.

#### Video

- [Video of the template working in action](https://drive.google.com/file/d/1-gnC1LKPtj35zbCazvlQrDSedEQtyeWD/view?usp=sharing)

### CODE REVIEW

**Review by Siumauricio — APPROVED** [review]
Thanks!!

**Comment by Siumauricio** [comment]
run `pnpm check` to format the files

File: apps/dokploy/templates/templates.ts [hunk 1]
```diff
@@ -512,8 +512,22 @@ export const templates: TemplateData[] = [
...
+		logo: "filebrowser.svg",
+		links: {
+			github: "https://github.com/filebrowser/filebrowser",
+			website: "https://filebrowser.org/",
+			docs: "https://filebrowser.org/",
+		},
+		tags: ["file", "manager"],
+		load: () => import("./filebrowser/index").then((m) => m.generate),
+  },
+  {
```
Comment (mezotv) [inline apps/dokploy/templates/templates.ts:530]:
```suggestion
	},
	{
```
try this @xmahbub

**Comment by xmahbub** [comment]
Thank you [Siumauricio](https://github.com/Siumauricio) . Format issue fixed ♥️

**Pull request merged**